### DISCUSSION TITLE
Leaving net8.0-android target in the TargetFrameworks causes ResolvePackageAssets task to fail

### DISCUSSION THREAD

**Discussion by trydalch** [unoplatform/uno#18047] [Q&A] [answered]
My teammate created a new project in VS 2022. He added our old uno project to it and began moving files into the new project (the old project had an issue where if we tried to change the Application Display Version, it wouldn't build). He then committed and pushed, and then I pulled the changes down. I opened the new solution, and was immediately faced with these errors. 

![image](https://github.com/user-attachments/assets/5ab3cf4a-2381-4647-b09a-70243e7b796f)

![image](https://github.com/user-attachments/assets/9602a155-4c37-48fa-810f-b6b08b033154)

If I remove it net8.0-android from the TargetFrameworks in the new project, all of those errors go away and packages are successfully restored for the other targets. The project then runs successfully. This also seems to be an issue that's local to my environment. 

I've tried deleting the .nuget/packages cache in my user directory, I've re-cloned the code. Same problem. It still builds and runs on my teammates pc. 

In the obj/project.assets.json file, there's a null in the projectFileDependencyGroups.net8.0-android34.0 list. 
``` 
"projectFileDependencyGroups": {
    "net8.0-android34.0": [
      null,
      "Microsoft.Extensions.Logging.Console >= 8.0.0",
      "Microsoft.NET.ILLink.Tasks >= 8.0.8",
      "SkiaSharp.Skottie >= 2.88.8",
      "SkiaSharp.Views.Uno.WinUI >= 2.88.8",
      "Uno.Dsp.Tasks >= 1.4.0",
      "Uno.Extensions.Configuration >= 4.2.2",
      "Uno.Extensions.Core.WinUI >= 4.2.2",
      "Uno.Extensions.Hosting.WinUI >= 4.2.2",
      "Uno.Extensions.Http.Refit >= 4.2.2",
      "Uno.Extensions.Http.WinUI >= 4.2.2",
      "Uno.Extensions.Localization.WinUI >= 4.2.2",
      "Uno.Extensions.Logging.WinUI >= 4.2.2",
      "Uno.Extensions.Navigation.Toolkit.WinUI >= 4.2.2",
      "Uno.Extensions.Navigation.WinUI >= 4.2.2",
      "Uno.Extensions.Reactive.Messaging >= 4.2.2",
      "Uno.Extensions.Reactive.WinUI >= 4.2.2",
      "Uno.Extensions.Serialization.Http >= 4.2.2",
      "Uno.Extensions.Serialization.Refit >= 4.2.2",
      "Uno.Fonts.OpenSans >= 2.4.0",
      "Uno.Material.WinUI >= 5.1.7",
      "Uno.Resizetizer >= 1.5.2",
      "Uno.Toolkit.WinUI >= 6.1.8",
      "Uno.Toolkit.WinUI.Material >= 6.1.8",
      "Uno.UI.Adapter.Microsoft.Extensions.Logging >= 5.3.123",
      "Uno.UniversalImageLoader >= 1.9.37",
      "Uno.WinUI >= 5.3.123",
      "Uno.WinUI.DevServer >= 5.3.123",
      "Uno.WinUI.Lottie >= 5.3.123",
      "Xamarin.AndroidX.Activity >= 1.8.2.1",
      "Xamarin.AndroidX.AppCompat >= 1.6.1.7",
      "Xamarin.AndroidX.Browser >= 1.8.0",
      "Xamarin.AndroidX.Legacy.Support.V4 >= 1.0.0.23",
      "Xamarin.AndroidX.RecyclerView >= 1.3.2.2",
      "Xamarin.AndroidX.SwipeRefreshLayout >= 1.1.0.18",
      "Xamarin.Google.Android.Material >= 1.10.0.3"
    ],
...
}
```

Anyone have any ideas? I'm at a loss.

**Comment by MartinZikmund** [comment, marked as answer]
Can you please run `uno-check` to verify everything is passing? Are you and your teammate both using the same version of .NET (`dotnet --version`)? Is it possible that there are some files that weren't pushed into the repository, but are present on your teammate's PC - it may be worth trying if they are able to clone the repo into a separate folder as well and then open successfully

**Reply by jeromelaban** [reply]
This also generally means that there's a csproj authoring error, or some bug in msbuild itself. Could you try moving your project to a shorter path, as well?

**Reply by trydalch** [reply]
Thank you both. The uno projects were nested inside a very large repository with a .gitignore that ignored the necessary XML files.
